lib/facade_converter.ts

```diff
@@ -303,6 +303,14 @@ export class FacadeConverter extends base.TranspilerBase {
       });
       this.emit(')');
     },
+    'Array.join': (c: ts.CallExpression, context: ts.Expression) => {
+      this.visit(context);
+      if (c.arguments.length === 0) {
```

Comment:
nit: I'm ok with writing `if (c.arguments.length) { ... }`, that's idiomatic enough in my opinion. YMMV.